layouts/shortcodes/observability_pipelines/processors/dedupe.md

```diff
@@ -1,7 +1,7 @@
 The dedupe processor removes copies of data to reduce volume and noise. It caches 5000 messages at a time and compares your incoming logs traffic against the cached messages. For example, this processor can be used to keep only unique warning logs in the case where multiple identical warning logs are sent in succession.
 
 To set up the deduplicate processor:
-1. Define a **filter query**. Only logs that match the specified [filter query](#filter-query-syntax) are checked for deduplication based on the rules below.
+1. Define a **filter query**. Only logs that match the specified [filter query](#filter-query-syntax) are processed. All logs, regardless of whether they do or do not match the filter query, are sent to the next step in the pipeline.
```

Comment:
Do you think a reader will be confused that dedupes will also be sent to the next step? I know its obvious to us but trying to be as specific as possible. eg 
Output of the dedupe filter and logs that do not match the filter query will be sent to the next step in the pipeline.

Reply:
if you make changes here, i think it will have to propagate to similar instances below.

Reply:
DM'd. All good here.